### PR TITLE
Began fiddling UI to match mocks

### PR DESCRIPTION
**Pull request by Malvolio**

### CODE REVIEW

File: frontend/src/PatientApp.tsx [hunk 1]
```diff
@@ -29,6 +27,20 @@ const styles = (theme: Theme) =>
     },
     title: {
       flexGrow: 1,
+      fontFmily: "Roboto",
```
Comment (tstanis-snickerbockens) [inline frontend/src/PatientApp.tsx:30]: not sure we should be switching font family as Susanne is not designing for style right now but more for function.

File: frontend/src/PatientApp.tsx [hunk 2]
```diff
@@ -29,6 +27,20 @@ const styles = (theme: Theme) =>
...
+      fontStyle: "normal",
+      fontWeight: "bold",
+      fontSize: "18px",
+      letterSpacing: "0.03em",
+      fontVariant: "small-caps",
+    },
+    actionButton: {
+      backgroundColor: "#FCD446",
```
Comment (tstanis-snickerbockens) [inline frontend/src/PatientApp.tsx:38]: is there a standard material color we can use?

File: frontend/src/PatientApp.tsx (hunk 1 above)
Comment (Malvolio) [reply frontend/src/PatientApp.tsx:30]: I just pulled the styles from Susanne's mock.

File: frontend/src/PatientApp.tsx (hunk 2 above)
Comment (Malvolio) [reply frontend/src/PatientApp.tsx:38]: Dunno, but Susanne is looking into MUI.

**Pull request merged**